projects/vdk-core/tests/vdk/internal/builtin_plugins/ingestion/test_ingester_base.py

```diff
@@ -27,12 +26,21 @@ def create_ingester_base() -> IngesterBase:
     }
     test_config = Configuration(None, config_key_value_pairs, {})
 
-    return IngesterBase(
-        data_job_name="test_job",
-        op_id="42a420",
-        ingester=MagicMock(spec=IIngesterPlugin),
-        ingest_config=IngesterConfiguration(test_config),
-    )
```

Comment:
```**(kwargs or {})```
?
then remove if-else redundant

Reply:
Doesn't work, IngesterBase expects either pre-processors and post-processors, or nothing. An empty dictionary is still considered something by python, so using `**(kwargs or {})` throws an exception.